### ISSUE TITLE
SSL-certificate expired

### ISSUE THREAD

**Issue by brean** [flynneva/flynn-labs#142]
Hey,

just wanted to let you know, no modern browser allows connecting to https://flynnlabs.dev/ because your SSL-Certificate expired.

**Comment by flynneva** [comment]
Yeah I've been meaning to get to that. www.Flynnlabs.dev still works though! 😅
